.github/workflows/golangci-lint.yaml

```diff
@@ -21,4 +21,4 @@ jobs:
       - name: golangci-lint
         uses: golangci/golangci-lint-action@3a919529898de77ec3da873e3063ca4b10e7f5cc # v3.7.0
         with:
-          version: v1.55.1
+          version: v1.55.2
```

Comment:
I _think_ @cpanato told me once that if you drop the `.2` altogether it will automatically pick up new patch versions. Just something to consider! Not blocking.

Reply:
drop the patch version and use just the major.minor, then it will always get the latest for the minor

also pleas eupdate to `v1.57`

Reply:
Fixed and force-pushed!